codecov.yml

```diff
@@ -0,0 +1,19 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 20%
```

Comment:
20! Man, so lenient. Don't want to keep yourself thinking about how to improve code coverage more than that?